internal/services/firewall/firewall_resource.go

```diff
@@ -224,6 +224,31 @@ func resourceFirewall() *pluginsdk.Resource {
 				},
 			},
 
+			"autoscale_configuration": {
+				Type:        pluginsdk.TypeSet,
+				Description: "Properties to provide a custom autoscale configuration to this azure firewall.",
+				Optional:    true,
+				Required:    false,
+				Elem: &schema.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"min_capacity": {
+							Type:         pluginsdk.TypeInt,
+							Description:  "The minimum number of capacity units for this azure firewall. Use null to reset the value to the service default.",
```

Comment:
The description can be removed as this provider doesn't use it to generate the document.